pubsub/protocol.py

```diff
@@ -1,20 +1,34 @@
 from pubsub.serializers.serializer import JSONSerializer
+from pubsub.validators.validator import ValidationError
+
+class ValidationErrorError(Exception):
+    def __init__(self, errors=None, *args, **kwargs):
+        self.errors = errors
+        super(ValidationErrorError, self).__init__(*args, **kwargs)
 
 
 class Protocol(object):
     """
     Protocol used to instantiate publisher/subscriber adapters with optional parameters(serializer, validator, filter)
     """
 
-    def __init__(self, adapter, serializer=None, validator=None, filter=None):
+    def __init__(self, adapter, serializer=None, validator=None, validation_error_handler=None, filter=None):
         self.adapter = adapter
         self.serializer = serializer or JSONSerializer()
         self.validator = validator
+        self.validation_error_handler = validation_error_handler
         self.filter = filter
 
-    def publish(self, topic, message):
+    def publish(self, topic, message, validation_error_message=False):
```

Comment:
Bit nitpicky, but I would call `validation_error_message` something else like `suppress_error_handler`.

Reply:
Yeah, then it becomes more general though, which isn't the worst.